docs/en/migration.md

```diff
@@ -1 +1,511 @@
+        type=val_dataset_type,  # Type of dataset
+        lq_folder='data/val_set5/Set5_bicLRx2',  # Path for lq folder
+        gt_folder='data/val_set5/Set5_mod12',  # Path for gt folder
+        pipeline=test_pipeline,  # See above for test_pipeline
+        scale=scale,  # Scale factor for upsampling
+        filename_tmpl='{}'))  # filename template
+```
+
+</td>
+
+<td valign="top">
+
+```python
+dataset_type = 'BasicImageDataset'  # The type of dataset
+data_root = 'data'  # Root path of data
+train_dataloader = dict(
+    num_workers=4,  # The number of workers to pre-fetch data for each single GPU
+    persistent_workers=False,  # Whether maintain the workers Dataset instances alive
+    sampler=dict(type='InfiniteSampler', shuffle=True),  # The type of data sampler
+    dataset=dict(  # Train dataset config
+        type=dataset_type,  # Type of dataset
+        ann_file='meta_info_DIV2K800sub_GT.txt',  # Path of annotation file
+        metainfo=dict(dataset_type='div2k', task_name='sisr'),
+        data_root=data_root + '/DIV2K',  # Root path of data
+        data_prefix=dict(  # Prefix of image path
+            img='DIV2K_train_LR_bicubic/X2_sub', gt='DIV2K_train_HR_sub'),
+        filename_tmpl=dict(img='{}', gt='{}'),  # Filename template
+        pipeline=train_pipeline))
+val_dataloader = dict(
+    num_workers=4,  # The number of workers to pre-fetch data for each single GPU
+    persistent_workers=False,  # Whether maintain the workers Dataset instances alive
+    drop_last=False,  # Whether drop the last incomplete batch
+    sampler=dict(type='DefaultSampler', shuffle=False),  # The type of data sampler
+    dataset=dict(  # Validation dataset config
+        type=dataset_type,  # Type of dataset
+        metainfo=dict(dataset_type='set5', task_name='sisr'),
+        data_root=data_root + '/Set5',  # Root path of data
+        data_prefix=dict(img='LRbicx2', gt='GTmod12'),  # Prefix of image path
+        pipeline=test_pipeline))
+test_dataloader = val_dataloader
+```
+
+</td>
+
+</tr>
+</thead>
+</table>
+
+### Evaluation settings
+
+We update evaluation settings in MMEdit 1.x. Important modifications are as following.
+
+- The evaluation field is split to `val_evaluator` and `test_evaluator`. The `interval` is moved to `train_cfg.val_interval`.
+- The metrics to evaluation are moved from `test_cfg` to `val_evaluator` and `test_evaluator`.
+
+<table class="docutils">
+<thead>
+  <tr>
+    <th> Original </th>
+    <th> New </th>
+<tbody>
+<tr>
+<td valign="top">
+
+```python
+train_cfg = None  # Training config
+test_cfg = dict(  # Test config
+    metrics=['PSNR'],  # Metrics used during testing
+    crop_border=scale)  # Crop border during evaluation
+
+evaluation = dict(  # The config to build the evaluation hook
+    interval=5000,  # Evaluation interval
+    save_image=True,  # Save images during evaluation
+    gpu_collect=True)  # Use gpu collect
+```
+
+</td>
+
+<td valign="top">
+
+```python
+val_evaluator = [
```

Comment:
The old and new code examples should be identical in function and meaning. For example, if there is a metric in the left, there should be only one metric in the new code. You may check if other examples also have this problem